### ISSUE TITLE
1st step in menu-bar reorganization: rename and relocate "Star Data" 

### ISSUE THREAD

**Issue by cegannon1** [ljramones/trips#124]
humans from left-to-right reading cultures have now largely been trained to expect to look at the option in the upper left hand corner of any menus bar to find the "start" or "load" commands.
However, in TRIPS, that function is really under "star data" which is two buttons further to the right. And "Star Data" sounds more like a catalog.
Yes, I know: it *is* a catalog. But most users, knowing or expecting what this program will do, will consider that of secondary importance. so what I would suggest is

1) rename Star Data, "Load Stars!"

2. Move that button to the first place along the radio bar.

That makes various manipulations, gathered under "General", located in a now very logical place. First I get the stars up, then I move immediately to the right to start massaging what I've got. 
Right now, the logical starting place (first in the upper left hand corner) is home to a menu button for a pull down menu that won't do ANYTHING. Not until the user (perhaps by chance/trial-and-error) fumbles onto the *third* button and loads a dataset. Which, in itself, is a far more cumbersome process than what most (including CHV) use. Which is essentially: load what database? and it throws you a list to click upon.

**Comment by ljramones** [comment]
this will need more definition. Currently, its isn't concrete enough

**Comment by ljramones** [comment]
done so far
